content/en/synthetics/browser_tests/advanced_options.md

```diff
@@ -43,7 +43,11 @@ To use a specific identifier (for example, to click on the `nth` element in a dr
 1. Record or manually add a [step][1] to your recording.
 2. Click on the recorded step and click **Advanced options**.
 3. Enter an XPath 1.0 selector or CSS class/ID under **User Specified Locator**, for example: `div`, `h1`, or `.hero-body`, for the HTML element.
-4. Once you have defined an element, click **Test** to highlight the element in the recording to the right.
+4. Optionally, handlebars syntax can be used to insert dynamic content such as variables:
+
+{{< img src="synthetics/browser_tests/advanced_options/advanced_user_locator.png" alt="User specified locator field highlighting handlebar syntax with variables" style="width:70%">}}
```

Comment:
Can we provide an example of what a `{{variable}}` would look like in this? It's clear from the UI that it is supported, but a user might want to read about _how_ to use it.

Reply:
Yes good catch! Let me re-work this a bit to provide an example :)

Reply:
Hi @kayayarai thanks for the suggestions! I re-worked this and added different screen shots to make this more clear, can you re-review next week?